README.md

```diff
@@ -97,7 +97,7 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 * Join the Ansible forum:
     * [ClickHouse Team](https://forum.ansible.com/g/ClickHouseTeam): by joining the team you will automatically get subscribed to the posts tagged with [clickhouse](https://forum.ansible.com/tag/clickhouse).
     * [Get Help](https://forum.ansible.com/c/help/6/none): get help or help others.
-    * [Posts tagged with 'postgresql'](https://forum.ansible.com/tag/clickhouse): leverage tags to narrow the scope.
+    * [Posts tagged with 'clickhouse'](https://forum.ansible.com/tag/clickhouse): leverage tags to narrow the scope.
```

Comment:
```suggestion
    * [Posts tagged with 'clickhouse'](https://forum.ansible.com/tag/clickhouse): use tags to narrow the scope.
```

Reply:
IMO "leverage" is a bit too much when you can simply say "use".

Reply:
simplicity is good, thanks, ready for review